core/src/main/java/io/undertow/server/handlers/cache/ResponseCachingStreamSinkConduit.java

```diff
@@ -167,4 +167,4 @@ public long writeFinal(ByteBuffer[] srcs, int offset, int length) throws IOExcep
     public int writeFinal(ByteBuffer src) throws IOException {
         return Conduits.writeFinalBasic(this, src);
     }
-}
+}
```

Comment:
I have no idea why this pops up, Ive set this file to be literal copy of one contained in previous commit yet this change somehow persists.